data/changelog.md

```diff
@@ -1,3 +1,6 @@
+* **Enhancement:** Centralize word generators to one module 
+  (`world/vocabulary`) and provide a simple interface (`ListGenerator`) for 
+  creating new word generators.
```

Comment:
I don't generally update the changelog for non-user-facing work. That's why that particular test isn't required. Might be worth adding an `**Internal**` tag or something, which is still broadcast to Discord but is ignored by the UI.

Reply:
Ah, that makes sense! I'll remove this. :sweat_smile: